requirements.txt

```diff
@@ -14,4 +14,4 @@ argcomplete >= 1.9.3, < 2
 commonmark >= 0.7.4, < 0.8
 docutils >= 0.14, < 1
 PyJWT >= 1.6.4
-retrying
+retrying==1.3.3
```

Comment:
@kislyuk  had some concerns about using this package in the query service "If possible, we should stick to configuring the retry logic already embedded in our dependencies, instead of bringing in another one (I've never used retrying, but from a brief glimpse at its issue tracker, it appears unmaintained, with the last release in 2014 and a fork that is recommended to replace it)." I'm not sure if those concerns apply here